helix-term/src/ui/popup.rs

```diff
@@ -128,6 +130,14 @@ impl<T: Component> Popup<T> {
         }
     }
 
+    /// Enables the Popup's scrollbar.
+    /// Consider disabling the scrollbar in case the child
+    /// already has its own.
+    pub fn with_scrollbar(mut self, enable_scrollbar: bool) -> Self {
+        self.has_scrollbar = enable_scrollbar;
+        self
+    }
```

Comment:
I think the completion and code-actions popups will need to have this disabled since their components are menus